packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -314,8 +370,9 @@ class Auth {
             return this.pendingToken
         }
         const accessToken = this.get('access_token')
+        const isGuest = this.get('customer_type') === 'guest'
 
-        if (accessToken && !this.isTokenExpired(accessToken)) {
+        if (accessToken && this.isTokenValidForHybrid(accessToken, isGuest)) {
```

Comment:
Sorry @shethj , I did not realize this is how the `isTokenExpired` was called, i think we should not include the `hasSFRAAuthStateChanged` in the isTokenValid because we are checking for refresh_token in that func.  We can simply call it directly from here
```suggestion
        if (accessToken && !this.isTokenExpired(token) && !this.hasSFRAAuthStateChanged(isGuest)) {
```

Reply:
Continue with the comment above on the isHybrid flag, would having that flag make it easier to separate the logic for hybrid or normal flow apps? 🤔. A normal app consumes this library should not be aware of what does SFRAState mean imo

Reply:
I don't mind adding a isHybrid flag but if we do, we'll want to talk about what isHybrid defaults to and I would argue that we would want it to default to `true` rather than `false since most current projects that start on PWA Kit are existing customers migrating from SFRA/SiteGen.

Reply:
We decided on keeping the default state as hybrid. So we're not adding flags for isHybrid 
Customers looking for a standalone installation can just remove the `OCAPISessionURL` from `CommerceApiProvider` to prevent the sessions API being called and the refresh-token comparison will not affect standalone execution